spec/tapioca/generators/base_generator_spec.rb

```diff
@@ -0,0 +1,170 @@
+# typed: true
+# frozen_string_literal: true
+
+require "spec_helper"
+require "generator_spec"
+require "tapioca/generators/base_generator"
+
+module Tapioca
+  module Generators
+    class BaseGeneratorSpec < GeneratorSpec
+      describe "#rbi_header" do
+        it "displays when config.file_header is true" do
+          config = ConfigBuilder.from_options(:dsl, {})
+          generator = Tapioca::Generators::BaseGenerator.new(config)
```

Comment:
If the class is marked `abstract` I don't think we should instantiate it.

If we want to test the base services, it would be better to create a dummy subclass that we can instantiate:

```rb
class DummyGenerator < BaseGenerator; end
```